packages/nimble-components/src/table-column/icon/tests/table-column-icon.stories.ts

```diff
@@ -76,17 +78,22 @@ export const iconColumn: StoryObj<IconColumnTableArgs> = {
             style="${isChromatic() ? '--ni-private-spinner-animation-play-state:paused' : ''}"
         >
             <${tableColumnTextTag} field-name="firstName" >
-                Name
+                First name
+            </${tableColumnTextTag}>
+            <${tableColumnTextTag} field-name="lastName" >
+                Last name
             </${tableColumnTextTag}>
```

Comment:
Can we make Last Name the last column and give it a large fractional width?

Reply:
There is now an additional column ("gender") that is the last column. I didn't adjust its fractional width, but I think the example shows the behavior adequately. Let me know if you disagree with the new example and still think the fractional widths should be updated.